sample/httpx_client.py

```diff
@@ -0,0 +1,48 @@
+import httpx
+from httpx import *
+from msal.oauth2cli import HttpClient, Response
+
+
+class HttpxClient(HttpClient):
```

Comment:
What is the difference between this implementation and [that one](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/169/files#diff-ec25ae689492ba368f2d6952907acaf4R6)?